config/vlan.py

```diff
@@ -185,3 +185,45 @@ def del_vlan_member(db, vid, port):
 
     db.cfgdb.set_entry('VLAN_MEMBER', (vlan, port), None)
 
+#
+# 'static-anycast-gateway' group ('config vlan static-anycast-gateway ...')
+#
+@vlan.group(cls=clicommon.AbbreviationGroup, name='static-anycast-gateway')
+def static_anycast_gateway():
+    pass
+
+@static_anycast_gateway.command('add')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@clicommon.pass_db
+def add_vlan_sag(db, vid):
+    """Enable static-anycast-gatweay on VLAN interface"""
+    ctx = click.get_current_context()
+
+    log.log_info(f"'vlan static-anycast-gateway add {vid}' executing...")
```

Comment:
Can this command be changed to "config vlan <vid> static-anycast-gateway enable" and ""config vlan <vid> static-anycast-gateway disable"

Reply:
Changed the corresponding commands from add/del to enable/disable.